rbd/snapshot.go

```diff
@@ -147,6 +147,7 @@ func (snapshot *Snapshot) IsProtected() (bool, error) {
 	return c_is_protected != 0, nil
 }
 
+// this method is Deplecated
```

Comment:
The key word here should be ["deprecated"](https://en.wiktionary.org/wiki/deprecate). The new comment should occur below the existing comment so that the doc comment continues to conform to standard doc comment formatting as checked by the `revive` tool. We should refer to the new, preferred method. It should be a full sentence: start with a capital letter and end with a period character (.).

Reply:
Could this comment be of this format?
"Deplecated : Method is Moved to rbd.go Package. "

Reply:
Not really, sorry. I've tried to make a proper github suggestion - see below.